config/ingest.go

```diff
@@ -64,6 +64,10 @@ type KafkaConfiguration struct {
 	// in case of large clusters where changes are more frequent.
 	// This value must not be set to value lower than 10s.
 	TopicMetadataRefreshInterval pkgkafka.TimeDurationMilliSeconds
+
+	// Enable contexts for extensive debugging of librdkafka.
+	// See: https://github.com/confluentinc/librdkafka/blob/master/INTRODUCTION.md#debug-contexts
+	DebugContexts []string
```

Comment:
Is this an exhaustive list? Do we validate invalid values?

Reply:
Fixed.